src/sagemaker/workflow/_utils.py

```diff
@@ -479,10 +482,19 @@ def arguments(self) -> RequestType:
 
             request_dict = get_create_model_package_request(**model_package_args)
         # these are not available in the workflow service and will cause rejection
+        warn_msg_template = (
+            "'%s' is popped out from the pipeline definition "
```

Comment:
will `x will be override by the pipeline runtime` be easier to understand?

Reply:
Updated in new commit to 
```
        warn_msg_template = (
            "Popping out '%s' from the pipeline definition "
            "since it will be overridden in pipeline execution time."
        )
```
Still inform customers about the pop out behavior in the warning in case customer gets surprised when they check in the definition json and is not able to find the ModelPackageName